packages/api-page-builder-so-ddb-es/src/operations/block/fields.ts

```diff
@@ -0,0 +1,21 @@
+import { BlockDynamoDbFieldPlugin } from "~/plugins/definitions/BlockDynamoDbFieldPlugin";
```

Comment:
BTW let's be more consistent with the rest of the system and name this folder `pageBlocks`.

Reply:
Let's do both. So, rename the folder and also make sure we have the full `pageBlock` name in consts / types / etc. For example, `createBlockEntity` should be `createPageBlockEntity`.

Reply:
@adrians5j @SvenAlHamad 
Just in case, I want to ask - what to do with block categories code? (it is already merged into umbrella branch)
it is not starting with `page*` prefix (not it is "blockCategory", not a "pageBlockCategory")